.circleci/config.yml

```diff
@@ -47,7 +47,7 @@ jobs:
   regression-test:
     <<: *defaults
     docker:
-      - image: circleci/golang:1.16-node
+      - image: circleci/golang:1.17-node
```

Comment:
question: why are we running the go-and-node image? What are we using node for?

Reply:
We're not, I took that from Policy Engine but it might have been a brainless copy-paste